test.js

```diff
@@ -0,0 +1,38 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var TestUtils = require('react-addons-test-utils');
+var OnClickOutside = require('.');
+
+describe('react-onclickoutside', function() {
+
+  var OutsideClickComponent = React.createClass({
+    mixins: [OnClickOutside],
+
+    getInitialState: function() {
+      return {handleClickOutsideCalled: false};
+    },
+
+    handleClickOutside: function() {
+      this.setState({handleClickOutsideCalled: true});
+    },
+
+    render: function() {
+      return React.createElement('div');
+    }
+  });
+
+  it('should not call handleClickOutside when clicking the element', function() {
+    var element = React.createElement(OutsideClickComponent);
+    var component = TestUtils.renderIntoDocument(element);
+    ReactDOM.findDOMNode(component).dispatchEvent(new MouseEvent('mousedown'));
+    assert(!component.state.handleClickOutsideCalled);
+  });
+
+  it('should call handleClickOutside when clicking the document', function() {
+    var element = React.createElement(OutsideClickComponent);
+    var component = TestUtils.renderIntoDocument(element);
+    document.dispatchEvent(new MouseEvent('mousedown'));
```

Comment:
I set this up to use Chrome (and hopefully Firefox/IE later) instead of phantomjs so that the tests could be more realistic.  Not sure if you have a preference for the direction things go here.